### PR TITLE
8 files change

### PR DESCRIPTION
**Pull request by rohitx007**
## Checklist
 - [ ]  A link to the issue/user story that the pull request relates to
 - [ ]  How to recreate the problem without the fix
 - [ ]  Design of the fix
 - [ ]  How to prove that the fix works
 - [ ]  Automated tests that prove the fix keeps on working
 - [ ]  Documentation - any JSDoc, website, or Stackoverflow answers?

## Steps to Reproduce

1.
2.
3.
4.

## Existing issues

- [ ] [Stack Overflow issues](http://stackoverflow.com/tags/hyperledger-composer)
- [ ] [GitHub Issues](https://github.com/hyperledger/composer/issues)
- [ ] [Rocket Chat history](https://chat.hyperledger.org/channel/composer)

### CODE REVIEW

File: packages/composer-cli/lib/cmds/network/upgradeCommand.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ module.exports.builder = {
     card: { alias: 'c', required: true, description: 'The cardname to use to upgrade the network', type:'string'},
     networkName: { alias: 'n', required: true, description: 'Name of the business network to upgrade' },
     networkVersion: { alias: 'V', required: true, description: 'Version of the business network to upgrade to' },
-    option: { alias: 'o', required: false, describe: 'Options that are specific specific to connection. Multiple options are specified by repeating this option', type: 'string' },
+    option: { alias: 'o', required: false, describe: 'Options that are specific to connection. Multiple options are specified by repeating this option', type: 'string' },
```
Comment (rohitx007) [inline packages/composer-cli/lib/cmds/network/upgradeCommand.js]: Rohit Sharma <31184621+rohitx007@users.noreply.github.com>

File: packages/composer-website/jekylldocs/reference/composer.network.upgrade.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Options:
   --card, -c                 The cardname to use to upgrade the network  [string] [required]
   --networkName, -n          Name of the business network to upgrade  [required]
   --networkVersion, -V       Version of the business network to upgrade to  [required]
-  --option, -o               Options that are specific specific to connection. Multiple options are specified by repeating this option  [string]
+  --option, -o               Options that are specific to connection. Multiple options are specified by repeating this option  [string]
```
Comment (rohitx007) [inline packages/composer-website/jekylldocs/reference/composer.network.upgrade.md]: Signed-off-by: Rohit Sharma <rohitx007@users.noreply.github.com>

File: packages/composer-website/jekylldocs/reference/composer.network.start.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ Options:
   --networkName, -n                  Name of the business network to start  [required]
   --networkVersion, -V               Version of the business network to start  [required]
   --loglevel, -l                     The initial loglevel to set  [choices: "INFO", "WARNING", "ERROR", "DEBUG"]
-  --option, -o                       Options that are specific specific to connection. Multiple options are specified by repeating this option  [string]
+  --option, -o                       Options that are specific to connection. Multiple options are specified by repeating this option  [string]
```
Comment (rohitx007) [inline packages/composer-website/jekylldocs/reference/composer.network.start.md]: Signed-off-by: Rohit Sharma <rohitx007@users.noreply.github.com>

File: packages/composer-website/jekylldocs/reference/composer.network.install.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ Options:
   -v, --version              Show version number  [boolean]
   --archiveFile, -a  The business network archive file name  [string] [required]
   --card, -c         The cardname to use to install the network  [string] [required]
-  --option, -o       Options that are specific specific to connection. Multiple options are specified by repeating this option  [string]
+  --option, -o       Options that are specific to connection. Multiple options are specified by repeating this option  [string]
```
Comment (rohitx007) [inline packages/composer-website/jekylldocs/reference/composer.network.install.md]: Signed-off-by: Rohit Sharma <rohitx007@users.noreply.github.com>

File: packages/composer-website/jekylldocs/reference/composer.identity.issue.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ Identity Options
 Options:
   --help             Show help  [boolean]
   -v, --version      Show version number  [boolean]
-  --option, -o       Options that are specific specific to connection. Multiple options are specified by repeating this option  [string]
+  --option, -o       Options that are specific to connection. Multiple options are specified by repeating this option  [string]
```
Comment (rohitx007) [inline packages/composer-website/jekylldocs/reference/composer.identity.issue.md]: Signed-off-by: Rohit Sharma <rohitx007@users.noreply.github.com>

File: packages/composer-cli/lib/cmds/network/startCommand.js [hunk 6]
```diff
@@ -24,7 +24,7 @@ module.exports.builder = function (yargs) {
         networkName: { alias: 'n', required: true, description: 'Name of the business network to start' },
         networkVersion: { alias: 'V', required: true, description: 'Version of the business network to start' },
         loglevel: { alias: 'l', required: false, describe: 'The initial loglevel to set', choices : ['INFO', 'WARNING', 'ERROR', 'DEBUG']},
-        option: { alias: 'o', required: false, describe: 'Options that are specific specific to connection. Multiple options are specified by repeating this option', type: 'string' },
+        option: { alias: 'o', required: false, describe: 'Options that are specific to connection. Multiple options are specified by repeating this option', type: 'string' },
```
Comment (rohitx007) [inline packages/composer-cli/lib/cmds/network/startCommand.js]: Signed-off-by: Rohit Sharma <rohitx007@users.noreply.github.com>

File: packages/composer-cli/lib/cmds/network/installCommand.js [hunk 7]
```diff
@@ -21,7 +21,7 @@ module.exports.describe = 'Installs a business network archive to Hyperledger Fa
 module.exports.builder = {
     archiveFile: {alias: 'a', required: true, describe: 'The business network archive file name', type: 'string' },
     card: { alias: 'c', required: true, describe: 'The cardname to use to install the network', type:'string'},
-    option: { alias: 'o', required: false, describe: 'Options that are specific specific to connection. Multiple options are specified by repeating this option', type: 'string' },
+    option: { alias: 'o', required: false, describe: 'Options that are specific to connection. Multiple options are specified by repeating this option', type: 'string' },
```
Comment (rohitx007) [inline packages/composer-cli/lib/cmds/network/installCommand.js]: Signed-off-by: Rohit Sharma <rohitx007@users.noreply.github.com>

File: packages/composer-cli/lib/cmds/identity/issueCommand.js [hunk 8]
```diff
@@ -22,7 +22,7 @@ module.exports.builder =function (yargs) {
         'newUserId': { alias: 'u', required: false, describe: 'The user ID for the new identity', type: 'string' },
         'participantId': { alias: 'a', required: true, describe: 'The participant to issue the new identity to', type: 'string' },
         'issuer': { alias: 'x', required: false, describe: 'If the new identity should be able to issue other new identities', type: 'boolean' },
-        'option': { alias: 'o', required: false, describe: 'Options that are specific specific to connection. Multiple options are specified by repeating this option', type: 'string' },
+        'option': { alias: 'o', required: false, describe: 'Options that are specific to connection. Multiple options are specified by repeating this option', type: 'string' },
```
Comment (rohitx007) [inline packages/composer-cli/lib/cmds/identity/issueCommand.js]: Signed-off-by: Rohit Sharma <rohitx007@users.noreply.github.com>

**Comment by jt-nti** [comment]
Hi @rohitx007, thanks very much for the pull request. Unfortunately you need to add the DCO sign off text to commit messages, rather than as comments on the pull request. Also, it would be really nice to have a single commit for this kind of change, rather than one per file. Would you be able to squash your commits and amend the message to include the DCO sign off text? Thanks again for your contribution.

**Comment by rohitx007** [comment]
ok sure

**Pull request closed without merging**